tock/tock/settings/base.py

```diff
@@ -34,13 +34,13 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'uaa_client',
-    'tock',
-    'projects',
-    'hours',
-    'employees',
-    'organizations',
-    'api',
-    'utilization',
+    'tock.apps.TockAppConfig',
+    'projects.apps.ProjectsAppConfig',
+    'hours.apps.HoursAppConfig',
+    'employees.apps.EmployeesAppConfig',
+    'organizations.apps.OrganizationsAppConfig',
+    'api.apps.ApiAppConfig',
+    'utilization.apps.UtilizationAppConfig',
```

Comment:
Oh 👏 on the better app organization!

Reply:
Given than employees, organizations, etc, are the apps, this is actually a bit confusing to my eyes.